benches/perf_ci/micro/class_create.py

```
@@ -0,0 +1,27 @@
+# perf_ci axis: class object creation (type creation) and instantiation.
+import os
+
+N = int(os.environ.get("PERF_CI_N", "2000"))
+
+total = 0
+for i in range(N):
```

Ruff (0.16.1)

[warning] Loop control variable `i` not used within loop body

(B007)